### PR TITLE
Rename osc to oc in README

### PR DESCRIPTION
**Pull request by jhadvig**
@rhcarvalho PTAL
@kargakis thanks :kissing:

### CODE REVIEW

File: app/View/Layouts/default.ctp [hunk 1]
```diff
@@ -264,7 +264,7 @@ osc env dc/frontend MYSQL_PASSWORD=&lt;your database user's password&gt;
                   </p>
                   <p>
                   Note: If the database service is created in the same project as the frontend pod,
-                  the *_SERVICE_HOST and *SERVICE_PORT environment variables will be automatically
+                  the *_SERVICE_HOST and _*SERVICE_PORT environment variables will be automatically
```
Comment (rhcarvalho) [inline app/View/Layouts/default.ctp]: *_SERVICE_PORT

**Comment by kargakis** [comment]
:+1:

**Comment by rhcarvalho** [comment]
LGTM. Merging this and following up to match https://github.com/openshift/rails-ex/pull/21

File: README.md [hunk 2]
```diff
@@ -31,15 +31,15 @@ Note, you can follow along with the web console (located at https://ip-address:8
 
 #### build ####
 
-        $ osc start-build cakephp-example-1 --follow
+        $ oc start-build cakephp-example --follow
 
-You can alternatively leave off `--follow` and use `osc build-logs cakephp-example-1` where n is the number of the build (output of start-build).
+You can alternatively leave off `--follow` and use `oc build-logs cakephp-example-n` where n is the number of the build (output of start-build).
```
Comment (rhcarvalho) [inline README.md]: why change from 1 to n?

Comment (rhcarvalho) [inline README.md]: oh , nevermind... obvious

**Pull request merged**